doc/source/how-to-upgrade-to-flower-next.rst

```diff
@@ -0,0 +1,241 @@
+changes that require manual updates.
+
+``SuperNode``/|clientapp_link|_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Wrap your existing client with |clientapp_link|_ instead of launching it via
+  |startclient_link|_. Here's an example:
+
+.. code-block:: python
+
+    # Flower 1.8
+    def client_fn(cid: str):
+        return flwr.client.FlowerClient().to_client() 
+    
+    app = flwr.client.ClientApp(
+       client_fn=client_fn,
+    )
+
+    # Flower 1.7
+    if __name__ == "__main__":
+        flwr.client.start_client(
+           server_address="127.0.0.1:8080",
+           client=flwr.client.FlowerClient().to_client(),
+        )
+
+|serverapp_link|_
+~~~~~~~~~~~~~~~~~
+- Wrap your existing strategy with |serverapp_link|_ instead of starting the server
+  via |startserver_link|_. Here's an example:
+
+.. code-block:: python
+
+    # Flower 1.8
+    app = flwr.server.ServerApp(
+        config=config,
+        strategy=strategy,
+    )
+
+    # Flower 1.7
+    if __name__ == "__main__":
+        flwr.server.start_server(
+            server_address="0.0.0.0:8080",
+            config=config,
+            strategy=strategy,
+        )
+
+Deployment
+~~~~~~~~~~
+- Run the ``SuperLink`` with |flowernext_superlink_link|_ before running, in sequence,
+  |flowernext_clientapp_link|_ and |flowernext_serverapp_link|_. There is no need to
+  execute `client.py` and `server.py` as Python scripts.
+- Here's an example to start the server without HTTPS:
+
+.. code-block:: bash
+    
+    # Start a Superlink
+    $ flower-superlink --insecure
+
+    # In a new terminal window, start a long-running SuperNode
+    $ flower-client-app client:app --insecure
+
+    # In another terminal window, start a long-running SuperNode (at least 2 SuperNodes are required)
+    $ flower-client-app client:app --insecure
+
+    # In another terminal window, run the apps
+    $ flower-server-app server:app --insecure
+
+- Here's another example to start with HTTPS. Use the ``--certificates`` command line
+  argument to pass paths to (CA certificate, server certificate, and server private key).
+
+.. code-block:: bash
+
+    # Start a secure Superlink
+    $ flower-superlink --certificates \
+        <your-ca-cert-filepath> \
+        <your-server-cert-filepath> \
+        <your-privatekey-filepath>
+
+    # In a new terminal window, start a long-running secure SuperNode
+    $ flower-client-app client:app \
+        --root-certificates <your-ca-cert-filepath> \
+        --server 127.0.0.1:9092
+
+    # In another terminal window, start a long-running secure SuperNode (at least 2 SuperNodes are required)
+    $ flower-client-app client:app \
+        --root-certificates <your-ca-cert-filepath> \
+        --server 127.0.0.1:9092
+
+    # In another terminal window, run the apps
+    $ flower-server-app server:app \
+        --root-certificates <your-ca-cert-filepath> \
+        --server 127.0.0.1:9091
+
+Simulation
+~~~~~~~~~~
```

Comment:
Should we add a section on notebooks after this section?

Reply:
Good idea, I've updated it.